lib/models/food.js

```diff
@@ -20,6 +20,39 @@ function create(req) {
     });
   })
 }
+function find (id) {
+  return database.raw('SELECT * FROM foods WHERE id = ?', id)
+};
+
+function update(req) {
+  return new Promise( (resolve, reject) => {
```

Comment:
This outer promise allows us to handle errors and resolve the promise, so that we can return an object rather than a promise.